apis/fluentd/v1alpha1/tests/tools.go

```diff
@@ -222,14 +222,14 @@ apiVersion: fluentd.fluent.io/v1alpha1
 kind: Output
 metadata:
   name: fluentd-output-user1
-  namespace: kubesphere-logging-system
+  namespace: fluent
   labels:
     output.fluentd.fluent.io/enabled: "true"
     output.fluentd.fluent.io/user: "user1"
 spec: 
   outputs: 
   - elasticsearch:
-      host: elasticsearch-logging-data.kubesphere-logging-system.svc
+      host: elasticsearch-logging-data.fluent.svc
```

Comment:
we can keep the es namespace as it is before : elasticsearch-logging-data.kubesphere-logging-system.svc